services/cases-api/lambdas/deleteCase.js

```diff
@@ -6,9 +6,6 @@ import * as response from '../../../libs/response';
 import { decodeToken } from '../../../libs/token';
 import * as dynamoDb from '../../../libs/dynamoDb';
 
-/**
- * Lambda deleting case by id from DynamoDB table cases
- */
 export async function main(event) {
```

Comment:
In this case i think that the comment is valid, since the function name is `main`. Would of course be better to rename the function but then we break our consistency in the codebase